cupy/random/_generator_api.pyx

```diff
@@ -285,6 +288,49 @@ class Generator:
 
         return y
 
+    def gamma(self, shape, scale=1.0, size=None):
+        """Returns an array of samples drawn from a gamma distribution.
+
+        .. seealso::
+            - :func:`cupy.random.gamma` for full documentation
+            - :meth:`numpy.random.RandomState.gamma`
```

Comment:
- OK to link to `cupy.random.gamma` for the explanation of the distribution.
- `numpy.random.Generator.gamma` seems more suitable than `numpy.random.RandomState.gamma`.

Reply:
Yes, I wanted to fix all docstrings in another PR but I will fix this one in here